guidance/rust_code/src/graph/mod.rs

```diff
@@ -0,0 +1,346 @@
+use std::fs::File;
+use std::io::{prelude::*, BufReader, Seek, SeekFrom};
+
+#[derive(PartialEq, PartialOrd, Debug)]
```

Comment:
Is it valid for GPSPoint to derive PartialOrd? In what situation would you compare which point is "greater" than another point?

Reply:
Removed partial ord